public/angular/js/filters/replaceCT.js

```diff
@@ -0,0 +1,12 @@
+angular.module('Aggie')
+
+.filter('replaceCT', function() {
```

Comment:
Neat! Could you please just add a code comment here explaining WHY this is being done? Future devs will probably scratch their heads at this 🙃

Reply:
sure